api/envoy/config/listener/v3/listener.proto

```diff
@@ -256,16 +255,6 @@ message Listener {
   // emitted by this listener.
   repeated accesslog.v3.AccessLog access_log = 22;
 
-  // If the protocol in the listener socket address in :ref:`protocol
-  // <envoy_api_field_config.core.v3.SocketAddress.protocol>` is :ref:`UDP
-  // <envoy_api_enum_value_config.core.v3.SocketAddress.Protocol.UDP>`, this field specifies the actual udp
-  // writer to create, i.e. :ref:`name <envoy_api_field_config.core.v3.TypedExtensionConfig.name>`
-  //    = "udp_default_writer" for creating a udp writer with writing in passthrough mode,
-  //    = "udp_gso_batch_writer" for creating a udp writer with writing in batch mode.
-  // If not present, treat it as "udp_default_writer".
-  // [#not-implemented-hide:]
-  core.v3.TypedExtensionConfig udp_writer_config = 23;
```

Comment:
note: I just outright moved this vs. deprecation as this is IMO alpha / only used in QUIC and also IMO it's in the wrong place. Let me know if you prefer the real deprecation dance.

Reply:
This is legit, it was still marked not-implemented-hide IMHO, but happy to here from others.